### PR TITLE
man: Document all options for doveadm

### PR DESCRIPTION
**Pull request by cmouse**

### CODE REVIEW

**Review by sirainen — DISMISSED** [review]

File: source/man/global-options.inc.in [hunk 1]
```diff
@@ -6,11 +6,27 @@ Global :man:`doveadm(1)` *options*:
...
+**-c** *config-file*
+   Read configuration from the given *config-file*. By default
+   it first reads config socket, and then falls back to
+   *@pkgsysconfdir@/dovecot.conf*. You can also point this to
+   config socket of some instance running compatible version.
+
+**-i** *instance-name*
+   If using multiple Dovecot instances, choose the config file based on this instance name.
+   See :dovecot_core:ref:`instance_name` setting for more information.
+
```
Comment (fleischie) [inline source/man/global-options.inc.in:18]: Would it be possible to match the order of the options here with the SYNOPSIS?

So `[-DOkv]` but then `-c`, `-f`, `-i`, and only then `-o`. I was a bit confused that I didn't find the option where I was looking for it. Feel free to disagree and resolve this comment unchanged. :)

**Review by fleischie — CHANGES_REQUESTED** [review]

File: source/man/doveadm-acl.1.rst.in [hunk 2]
```diff
@@ -12,7 +12,7 @@ doveadm-acl - Manage Access Control List (ACL)
 SYNOPSIS
 ========
 
-**doveadm** [**-Dv**] [**-f** *formatter*] **acl** *command* [*OPTIONS*] [*ARGUMENTS*]
+**doveadm** [*global options*] **acl** *command* [*OPTIONS*] [*ARGUMENTS*]
```
[the remark this answers is not in the document]
Comment (cmouse) [reply source/man/doveadm-acl.1.rst.in:15]: i suppose.

File: source/man/global-options-formatter.inc.in [hunk 3]
```diff
@@ -1,10 +1,4 @@
-GLOBAL OPTIONS
-==============
-
-Global :man:`doveadm(1)` *options*:
-
-**-D**
-   Enables verbosity and debug messages.
+.. include:: global-options.inc
```
[the remark this answers is not in the document]
Comment (cmouse) [reply source/man/global-options-formatter.inc.in:1]: um, no? why would it?

[the remark this answers is not in the document]
Comment (cmouse) [reply source/man/global-options-formatter.inc.in:1]: in this commit, global-options-formatter.inc will have same options as it had before, nothing changes. the next commit adds new options.

[the remark this answers is not in the document]
Comment (fleischie) [reply source/man/global-options-formatter.inc.in:1]: Ah ok. Excuse my fallacy, I must not have been thinking straight. :open_mouth:

**Review by fleischie — APPROVED** [review]

**Pull request merged**